BUILD.md

```diff
@@ -24,3 +24,17 @@ tito tag
 ```
 tito build --rpm
 ```
+
+## Build openshift-ansible container image
```

Comment:
I didn't find a reference in README.md, there's one in CONTRIBUTING.md that I'll update to mention both RPM and container image.